lib/trix/version.rb

```diff
@@ -1,3 +1,3 @@
 module Trix
-  VERSION = '0.9.5'
+  VERSION = '0.9.5'.freeze
```

Comment:
I'd say to remove this. Not huge advantage by freezing only one string in the whole project. Not huge perf gain by doing this.